install/kubernetes/helm/subcharts/gateways/values.yaml

```diff
@@ -108,6 +108,16 @@ istio-egressgateway:
   - name: egressgateway-ca-certs
     secretName: istio-egressgateway-ca-certs
     mountPath: /etc/istio/egressgateway-ca-certs
+  #### Advanced options ########
+  envVariables:
+    # Set this to "external" if and only if you want the egress gateway to
+    # act as a transparent SNI gateway that routes mTLS/TLS traffic to
```

Comment:
It would help a lot if this comment included the actual ServiceEntry sample :-), or even better if that was included in the helm templates, with a simpler setting in values.yaml.

For example: enableEgressSni true, and templates adding in the needed entry and env.

Reply:
+1 that would be nice.

Reply:
So I tried that. the thing is you can't do helm on values.yaml. In other words, I can do, if egresssni.true, do foobar in values.yaml, else do xyz in values.yaml..